docker-compose.yml

```diff
@@ -47,8 +47,8 @@ services:
       context: ./web/Web
       dockerfile: ./compose/production/postgres/Dockerfile
     volumes:
-      - /postgres_data_local:/var/lib/postgresql/data
-      - /postgres_backup_local:/backups
+      - ./postgres_data_local:/var/lib/postgresql/data
+      - ./postgres_backup_local:/backups
```

Comment:
Let's not worry about this for this pull request. Having the dot breaks things in my machine and on the tuna server.